src/index.js

```diff
@@ -24,6 +25,10 @@ class Dropzone extends React.Component {
 
   componentDidMount() {
     this.enterCounter = 0;
+    const { handleOnCancel } = this;
+    document.body.onfocus = (event) => {
```

Comment:
Also, instead of creating a clojure here `(event) => { handleOnCancel(event) }` you could just pass the  `this.handleOnCancel` as a listener since it has the same signature.